.gitignore

```diff
@@ -51,6 +51,7 @@ coverage.xml
 .hypothesis/
 .pytest_cache/
 cover/
+tests/input/*
```

Comment:
there are files checked in here as well, I think it would be confusing to ignore the path (and these files are useful to distribute for evaluation purposes)